benchmarks/benchmark_shared.c

```diff
@@ -42,6 +42,9 @@
 #include "shared.h"
 #include "benchmark_shared.h"
 
+#define FT_MIN_ITERS 10
+#define FT_BW_ITERS MAX(opts.iterations / MAX(opts.window_size / 8, 1), FT_MIN_ITERS)
```

Comment:
The MIN value will increase the time to run a quick test.  I would just do:

opts.iterations / opts.window_size